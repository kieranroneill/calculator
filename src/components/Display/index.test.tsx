import { shallow, ShallowWrapper } from 'enzyme';
import * as React from 'react';

// Components.
import { Display, Props } from './';

interface Scope {
    props: Props;
    wrapper: ShallowWrapper;
}

describe('src/components/Display', () => {
    let scope: Scope;

    beforeEach(() => {
        const props: Props = {
            value: 'Testing cannot continue until your Companion Cube has been incinerated.',
        };

        scope = {
            props,
            wrapper: shallow(<Display {...props} />),
        };
    });

    describe('<Display /> snapshots', () => {
        it('should match the snapshot with the default props', () => {
            expect(scope.wrapper).toMatchSnapshot();
        });
    });
});
