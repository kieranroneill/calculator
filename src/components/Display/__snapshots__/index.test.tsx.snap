// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Display <Display /> snapshots should match the snapshot with the default props 1`] = `
<styled.div>
  <styled.input
    readOnly={true}
    value="Testing cannot continue until your Companion Cube has been incinerated."
  />
</styled.div>
`;
